nbrsessionproxy/handlers.py

```diff
@@ -67,17 +58,18 @@ class RSessionProxyHandler(IPythonHandler):
 
     def initialize(self, state):
         self.state = state
+        self.port = 9797
```

Comment:
This prevents more than one user from running this from non-container spawners.

Reply:
Yup, good catch. Updated to be dynamic again, but a little cleaner.